### ISSUE TITLE
The details about ProteinForceGPT

### ISSUE THREAD

**Issue by Guardian-in-the-WF** [lamm-mit/ProtAgents#1]
Could you provide some details about the pretraining of ProteinForceGPT model. If I want to finetune downstream tasks by using ProteinForceGPT, how can I process the inputs and labels? Looking for your reply. Thank you very much.
